default/scripting/buildings/colonies/col_bld_gen.py

```diff
@@ -211,19 +212,22 @@
     sp_id = species[0]
     sp_name = sp_id.split("_", 1)[1]
     sp_desc_name = species[1]
+    sp_tags = '"'+sp_id+'"'
```

Comment:
We try to follow PEP 0008, so please add spaces around the `+` operator.

[PEP 0008 - Other recommendations][https://www.python.org/dev/peps/pep-0008/#other-recommendations]